code/datums/gamemodes/gangwar.dm

```diff
@@ -1215,10 +1216,16 @@ proc/broadcast_to_all_gangs(var/message)
 		. = ..()
 		. += "The screen displays \"Total Score: [gang.gang_score()]\""
 
-	attack_hand(var/mob/living/carbon/human/user)
+	attack_hand(var/mob/user)
 		if(!isalive(user))
 			boutput(user, SPAN_ALERT("Not when you're incapacitated."))
 			return
+		if(isghostcritter(user))
+			boutput(user, SPAN_ALERT("You can't understand the concept of gangs."))
+			return
```

Comment:
why isn't `NO_GHOSTCRITTER` enough

Reply:
wasn't aware there was code for NO_GHOSTCRITTER to handle large objects. as i'd only seen it used in items before. fixed